### PR TITLE
Set up MiMa.

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
Review by @sjrd

File: seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/AbstractSeleniumJSRunner.scala [hunk 1]
```diff
@@ -3,11 +3,17 @@ package org.scalajs.jsenv.selenium
 import org.scalajs.core.tools.io.{MemVirtualJSFile, VirtualJSFile}
 import org.scalajs.core.tools.jsdep.ResolvedJSDependency
 import org.scalajs.core.tools.logging.Logger
-import org.scalajs.jsenv.JSConsole
+import org.scalajs.jsenv.{JSConsole, VirtualFileMaterializer}
 
 abstract class AbstractSeleniumJSRunner(browserProvider: SeleniumBrowser,
     libs: Seq[ResolvedJSDependency], code: VirtualJSFile, materializer: FileMaterializer) {
 
+  @deprecated("Added FileMaterializer to the parameters. Falling back to DefaultFileMaterializer.", "0.1.2")
```
Comment (sjrd) [inline seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/AbstractSeleniumJSRunner.scala]: Consider the message

```
Use the overload with an explicit FileMaterializer.
```

which is the "format" of deprecation messages that we typically use.

File: build.sbt [hunk 2]
```diff
@@ -17,7 +33,37 @@ val commonSettings: Seq[Setting[_]] = Seq(
...
+    if (!scalaVersionsUsedForPublishing.contains(scalaV)) {
+      // This artifact will not be published. Binary compatibility is irrelevant.
+      None
+    } else if (newScalaBinaryVersionsInThisRelease.contains(scalaBinaryV)) {
+      // New in this release, no binary compatibility to comply to
+      None
+    } else {
+      val thisProjectID = projectID.value
+      val previousCrossVersion = thisProjectID.crossVersion match {
+        case ScalaJSCrossVersion.binary => previousBinaryCrossVersion
```
Comment (sjrd) [inline build.sbt]: This is a bit overkill for this project. It does not contain any Scala.js project, so `ScalaJSCrossVersion.binary` never happens. So this `match` is unnecessary, and so is `previousBinaryCrossVersion`.

File: build.sbt [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
 import org.scalajs.jsenv.selenium.SeleniumJSEnv
 import org.scalajs.jsenv.selenium.Firefox
 import org.scalajs.jsenv.selenium.CustomFileMaterializer
 
+import com.typesafe.tools.mima.plugin.MimaPlugin.mimaDefaultSettings
+import com.typesafe.tools.mima.plugin.MimaKeys.{previousArtifact, binaryIssueFilters}
+
+val previousVersion = "0.1.1"
+val previousSJSBinaryVersion =
+  ScalaJSCrossVersion.binaryScalaJSVersion(previousVersion)
```
Comment (sjrd) [inline build.sbt]: This does not make any sense. `binaryScalaJSVersion` expects a version number _of Scala.js_, not of an arbitrary library. Fortunately, you don't need it ;)

**Comment by sjrd** [comment]
That's all.

**Comment by sjrd** [comment]
LGTM

**Pull request merged**